main/perl-try-tiny/APKBUILD

```diff
@@ -1,36 +1,38 @@
 # Automatically generated by apkbuild-cpan, template 1
-# Contributor: Kiyoshi Aman <kiyoshi.aman@gmail.com>
-# Maintainer: Kiyoshi Aman <kiyoshi.aman@gmail.com>
```

Comment:
I think it would make sense to keep at least the Contributor comment.

Reply:
Done

Reply:
:ping_pong: @ncopa